### ISSUE TITLE
Email quick search bug

### ISSUE THREAD

**Issue by rutkas** [espocrm/espocrm#1161]
Email lookup (quick search is not working properly).
When entity has more then one email, for example primary email is a@a.com and another email in the entity is b@b.com. We try to search by one of SECONDARY emails in TO field of email compose view by typing b@b... and as a result lookup shows PRIMARY email a@a.com of this entity instead of email that is searched.
tested on the latest Version 5.5.1

**Comment by yurikuzn** [comment]
Does it show secondary email address below?

**Comment by yurikuzn** [comment]
I tested and it showed both email address.

**Comment by rutkas** [comment]
No, it does not show secondary (correct address below). It shows only one primary address instead of secondary that is being searched. I see you marked a bug label, were you able to reproduce it ?

**Comment by yurikuzn** [comment]
I didn't reproduced actually. I supposed the problem is that secondary email is not shown first if a query is matched.

**Comment by rutkas** [comment]
secondary emails are not shown at all
